tutorials/tutorial-one-dotnet.md

```diff
@@ -93,7 +93,7 @@ Then we add the client dependency.
 ```powershell
 cd Send
 dotnet add package RabbitMQ.Client
-cd /docs/Receive
+cd /../Receive
```

Comment:
Shouldn't it be `../Receive` (without the leading slash)?

Reply:
my apologies, you are absolutely right
should be fixed now